protobuf_deps.bzl

```diff
@@ -13,6 +13,7 @@ PROTOBUF_MAVEN_ARTIFACTS = [
     "com.google.truth:truth:1.1.2",
     "junit:junit:4.13.2",
     "org.mockito:mockito-core:4.3.1",
+    "org.jruby:jruby-complete:9.2.20.1"
```

Comment:
Same question as above, with regard to how we can continue to support testing of both JRuby 9.2.x and 9.3.x.